scripts/handle-translations.js

```diff
@@ -118,6 +117,11 @@ const runProcess = (data) => {
                 callback,
             )
             break
+        case 'merge-master-translation':
+            exec(
+                `git merge -X theirs --commit master -m "chore: $(git status --short)"`,
+                callback,
+            )
```

Comment:
do we need a break here @prince-deriv ? or we wanna fall through the default? 🙏

Reply:
yes we need a break.